### PR TITLE
Adding some more stuff to windows-debug image

### PR DESCRIPTION
**Pull request by marosset**
Adding some more debugging tools to `windows-debug` image

### CODE REVIEW

**Comment by marosset** [comment]
I verified the files make it into the debug image with `mrosse3/windows-debug:pr-14`

/assign @jsturtevant

File: images/Dockerfile [hunk 1]
```diff
@@ -6,3 +6,7 @@ WORKDIR /apps
 ENV PATH="%CONTAINER_SANDBOX_MOUNT_POINT%\apps\vim;%CONTAINER_SANDBOX_MOUNT_POINT%\apps\sysinternals;C:\Windows\system32;C:\Windows"
 
 COPY --from=installer /ProgramData/scoop/apps/vim/9.0 ./vim/
+
+WORKDIR /debug
```
Comment (jsturtevant) [inline images/Dockerfile:10]: could we set the WORKDIR back to the root after copying (or does COPY create the folder if it doesn't exist?

File: images/Dockerfile [hunk 2]
```diff
@@ -6,3 +6,7 @@ WORKDIR /apps
...
+COPY --from=installer /content/ContainerPlatform.wprp .
+COPY --from=installer /debug/net ./net
```
Comment (jsturtevant) [inline images/Dockerfile:12]:
```suggestion
COPY --from=installer /debug/net ./network
```

File: images/content/ContainerPlatform.wprp [hunk 3]
```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0"?>
```
Comment (jsturtevant) [inline images/content/ContainerPlatform.wprp:1]: maybe a readme how to use this?

File: images/Dockerfile (hunk 1 above)
Comment (marosset) [reply images/Dockerfile:10]: I think we can just drop the WORKDIR. Let me try that.

File: images/content/ContainerPlatform.wprp (hunk 3 above)
Comment (marosset) [reply images/content/ContainerPlatform.wprp:1]: added

**Pull request merged**